example_files/post-processing/R14_trend_plot.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import matplotlib
+matplotlib.use("agg")
+import matplotlib.pyplot as plt
+import seaborn
+c = seaborn.color_palette('colorblind')
+
+fig_width_pt = 750.  # Get this from LaTeX using \showthe\columnwidth
+inches_per_pt = 1. / 72.27  # Convert pt to inch
+golden_mean = (np.sqrt(5) - 1.) / 2.  # Aesthetic ratio
+fig_width = fig_width_pt * inches_per_pt  # width in inches
+fig_height = 0.9 * fig_width * golden_mean  # height in inches
+fig_size = [fig_width, fig_height]
+params = {'backend': 'pdf',
+          'axes.labelsize': 18,
+          'legend.fontsize': 18,
+          'xtick.labelsize': 18,
+          'ytick.labelsize': 18,
+          'text.usetex': True,
+          'font.family': 'Times New Roman',
+          'figure.figsize': fig_size}
+matplotlib.rcParams.update(params)
+
+# user specify input
+R14_true = 11.55
+label = 'ZTF'
+
+data_GWEM = pd.read_csv('../../output/Figures/GW_EM_R14trend_{0}.dat'.format(label), header=0, delimiter=' ')
```

Comment:
More command line arguments (and consider specifying all of thse at the top of the script).